lib/utils/dom.js

```diff
@@ -35,14 +35,17 @@ export function observe(ele, options, callback) {
 	return observer;
 }
 
-export function waitForChild(ele, selector) {
+export function waitForChild(ele, selector, subtree = false) {
 	return new Promise(resolve => {
-		if (Array.from(ele.children).some(child => $(child).is(selector))) {
+		if (
+			(subtree && $(ele).find(selector).length) ||
+			(!subtree && Array.from(ele.children).some(child => $(child).is(selector)))
+		) {
 			resolve();
 			return;
 		}
 
-		const observer = observe(ele, { childList: true }, mutation => {
+		const observer = observe(ele, { childList: true, subtree }, mutation => {
```

Comment:
This is probably to expensive (especially if there's many new features). I'm looking for an alternative.